src/shared/components/home/site-form.tsx

```diff
@@ -76,6 +77,7 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
         application_question: site.application_question,
         private_instance: site.private_instance,
         default_theme: site.default_theme,
+        legal_information: site.legal_information,
```

Comment:
You'll notice this is the component that handles EditSite, not admin-settings.tsx